backend/handlers/handlers.go

```diff
@@ -632,53 +639,66 @@ func (handlers *Handlers) getConvertToFiatHandler(r *http.Request) (interface{},
 	from := r.URL.Query().Get("from")
 	to := r.URL.Query().Get("to")
 	amount := r.URL.Query().Get("amount")
-	amountAsFloat, err := strconv.ParseFloat(amount, 64)
-	if err != nil {
+	plain := r.URL.Query().Get("plain")
+	amountRat, valid := new(big.Rat).SetString(amount)
+	if !valid {
 		return map[string]interface{}{
 			"success": false,
 			"errMsg":  "invalid amount",
 		}, nil
 	}
+
 	rate := handlers.backend.RatesUpdater().LatestPrice()[from][to]
+	convertedAmount := new(big.Rat).Mul(amountRat, new(big.Rat).SetFloat64(rate))
+	var formattedAmount string
+	if plain == "true" {
+		formattedAmount = coinpkg.FormatAsPlainCurrency(convertedAmount, to)
+	} else {
+		formattedAmount = coinpkg.FormatAsCurrency(convertedAmount, to)
+	}
```

Comment:
don't see this being used anywhere? why keep it? could get rid of the `plain` param

Reply:
I thought that in this way it can be used (now and in the future) to serve both kinds of formatting needs. I'll change it

Reply:
Should be ok now